src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ActiveSession/index.js

```diff
@@ -33,6 +34,24 @@ const ActiveSession = ({ classes, freeCallsRemaining, handleComplete, freeCallsA
         <span className={classes.FreeApiCallsText}>Free API Calls</span>
         <span className={classes.ReaminaingCallsNo}>{freeCallsRemaining}</span>
         <StyledLinearProgress value={progressValue()} />
+        <AlertBox
+          type={alertTypes.WARNING}
+          message={
+            <Fragment>
+              <span>
+                We have temporarily disabled this action as we are hard forking the AGI token to make it cross
```

Comment:
Please make it "cross chain compatible"

Reply:
Do we need a space between `more{" "}`

Reply:
updated https://github.com/singnet/snet-dapp/pull/756/commits/f2a81ad2de61a63ee049920318cccc6a1e654c9c